api/base/filters.py

```diff
@@ -218,7 +218,22 @@ def parse_query_params(self, query_params):
                                 'source_field_name': source_field_name
                             }
                         })
-
+                    elif source_field_name == 'is_preprint':
+                        op = 'eq' if value == 'false' else 'ne'
```

Comment:
There's a util called `is_truthy` that would allow things like `?filter[preprint]=0` to work in this case